### ISSUE TITLE
Some players are unable to connect,

### ISSUE THREAD

**Issue by ThousandX** [AscensionGameDev/Intersect-Engine#99]
**Describe the bug**
I am receiving a lot of reports from players with a status error (Unknown) in the status tab.

**Version Encountered**
Version 0.6.2.1 DEV

**To Reproduce**
Dont know why

**Screenshots**
![unknown](https://user-images.githubusercontent.com/21990960/78519024-687d5500-7798-11ea-83c6-58930270f2b7.png)

**Logs**
`2020-04-05 20:13:32.741 [Error] No handlers for 'OnConnectionDenied'.
2020-04-05 20:14:31.148 [Error] No handlers for 'OnConnectionDenied'.
2020-04-05 20:16:03.181 [Error] No handlers for 'OnConnectionDenied'.
2020-04-05 20:17:40.198 [Error] No handlers for 'OnConnectionDenied'.
2020-04-05 20:20:28.606 [Error] No handlers for 'OnConnectionDenied'.
2020-04-05 20:23:51.906 [Error] No handlers for 'OnConnectionDenied'.
2020-04-05 20:27:11.487 [Error] No handlers for 'OnConnectionDenied'.
2020-04-05 20:51:30.804 [Error] No handlers for 'OnConnectionDenied'.
2020-04-05 21:28:03.584 [Error] No handlers for 'OnConnectionDenied'.
2020-04-05 21:40:25.329 [Error] No handlers for 'OnConnectionDenied'.
2020-04-05 21:41:13.501 [Error] No handlers for 'OnConnectionDenied'.
2020-04-05 21:43:35.863 [Error] No handlers for 'OnConnectionDenied'.
2020-04-05 21:52:40.214 [Error] No handlers for 'OnConnectionDenied'.
2020-04-05 22:03:54.598 [Error] No handlers for 'OnConnectionDenied'.
2020-04-05 22:09:31.603 [Error] No handlers for 'OnConnectionDenied'.
2020-04-05 22:14:30.122 [Error] No handlers for 'OnConnectionDenied'.
2020-04-05 22:17:18.875 [Error] No handlers for 'OnConnectionDenied'.
2020-04-05 22:22:10.332 [Error] No handlers for 'OnConnectionDenied'.
2020-04-05 22:38:11.423 [Error] No handlers for 'OnConnectionDenied'.
2020-04-05 22:47:40.525 [Error] No handlers for 'OnConnectionDenied'.
2020-04-05 22:52:02.653 [Error] No handlers for 'OnConnectionDenied'.
2020-04-05 23:04:42.851 [Error] No handlers for 'OnConnectionDenied'.
2020-04-05 23:06:17.962 [Error] No handlers for 'OnConnectionDenied'.
2020-04-05 23:18:34.644 [Error] No handlers for 'OnConnectionDenied'.
2020-04-05 23:19:11.738 [Error] No handlers for 'OnConnectionDenied'.
2020-04-05 23:23:17.507 [Error] No handlers for 'OnConnectionDenied'.`

**Operating System:**
Windows Server 2016

**Comment by jcsnider** [comment]
Should be fixed once #100 is merged into development

**Comment by jcsnider** [comment]
Panda just merged it. Go ahead and pull latest and give it a try.
